PowerShell Scanners/Important Events/Important Events.ps1

```diff
@@ -0,0 +1,26 @@
+
+[CmdletBinding()]
+param (
+    # Number of days worth of entries to collect
+    [int]$Days = 28, # Defaults to Four weeks
```

Comment:
Could you change this to be `[uint]`? An `[int]` allows for negative numbers to be passed would opens the chance of a double negative cast for `$StartDate` giving you a positive number and then Get-WinEvent will try and search for events in the future. A `[uint]` will only ever take positive numbers ensuring that the double negative cast doesn't occur.

Reply:
Done.